Update 2023.7 release blog with hyperion breaking change (#28125)
home-assistant/home-assistant.github.io: source/_posts/2023-07-05-release-20237.markdown

{% enddetails %}

{% details "Hyperion" %}

> The behavior of hyperion lights has changed:

Reviewer: ```suggestion
The behavior of Hyperion lights has changed:
```

> Turning off the light entity will only clear the priority you have configured during integration setup. Other HA-independent light sources in Hyperion might still be active and cause light to be emitted. In order to turn the light output off entirely regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. Subsequently, external effects that are unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.

Reviewer: ```suggestion
Turning off the light entity will only clear the priority you have configured during the integration setup. Other Home Assistant independent light sources in Hyperion might still be active and cause light to be emitted. If you want to turn the light off entirely, regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. 

External effects that are unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.
```

Reviewer: do you think we could delete the phrase _regardless of active light sources_? I find it a bit confusing.

Author: @sab44 Can you review the suggestion from @c0ffeeca7, I cant' judge if it's OK to remove?

Reviewer 2: @emontnemery @c0ffeeca7 I think we need this sentence the way it is to emphasize that the switch will ignore any active light sources. I agree that the wording could be improved but I'm unable to come up with better phrasing right now.

Reviewer: Thank you for clarifying.

> Turning off the light entity will only clear the priority you have configured during integration setup. Other HA-independent light sources in Hyperion might still be active and cause light to be emitted. In order to turn the light output off entirely regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. Subsequently, external effects that are unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.

Reviewer 3: ```suggestion
Turning off the light entity will only clear the priority you have configured during the integration setup. Other HA-independent light sources in Hyperion might still be active and cause light to be emitted. To turn the light output off entirely regardless of active light sources, you can enable the `switch.[instance]_led_device` entity that acts as a global switch. Subsequently, external effects unrelated to the Hyperion light source (such as USB capture) have been removed from the light entity. To control them, enable the respective disabled-by-default switch entity.
```
